go/src/koding/kites/kloud/stack/provider/apply.go

```diff
@@ -36,20 +37,22 @@ import (
 // is zeroed, which could make the destroy oepration to fail - we
 // first build machines and rest of the destroy is perfomed asynchronously.
 func (bs *BaseStack) HandleApply(ctx context.Context) (interface{}, error) {
-	var arg stack.ApplyRequest
-	if err := bs.Req.Args.One().Unmarshal(&arg); err != nil {
-		return nil, err
+	arg, ok := ctx.Value(stack.ApplyRequestKey).(*stack.ApplyRequest)
```

Comment:
In this PR I call `HandleBootstrap` method. The `HandleBootstrap` method before the change read BootstrapRequest value from `(*kite.Request).Args`, after the change it looks it up in the `context.Context` first and use it instead if found. The change is required to make it possible to call `HandleBoostrap` outside kite context.

Also I've applied the same change to all `Handle*` methods for the sake of consistency.